src/org/opendatakit/briefcase/ui/pull/PullPanel.java

```diff
@@ -22,7 +22,8 @@
 import static org.opendatakit.briefcase.ui.reused.UI.errorMessage;
 
 import java.util.Optional;
-import javax.swing.JPanel;
+import javax.swing.*;
```

Comment:
This makes checkstyle angry :P 

We have a rule that forbids using `.*` imports.

Reply:
Ok, I'm glad to be helpful, I will review my IDE configuration on imports next time. :)